packages/solid-crs-presentation/lib/app-root.component.ts

```diff
@@ -27,7 +27,7 @@ export class AppRootComponent extends RxLitElement {
    * The component's translator.
    */
   @property({ type: Object })
-  public translator: Translator = new MemoryTranslator('nl-NL');
+  public translator: Translator = new MemoryTranslator(new URL(window.location.href).searchParams.get('lang') || 'nl-NL');
```

Comment:
Isn't this a bit weird? It sets the default to whatever is first in the URL? Creating it with a static default (or env var) and then changing it immediately if there is already another language in the URL would be cleaner, I think.